Dockerfile

```diff
@@ -64,18 +64,16 @@ RUN wget https://bootstrap.pypa.io/get-pip.py && \
 # Clone TensoFlow from Mozilla repo
 RUN git clone https://github.com/mozilla/tensorflow/
 WORKDIR /tensorflow
-RUN git checkout r1.13
+RUN git checkout r1.14
 
 
 # GPU Environment Setup
 ENV TF_NEED_CUDA 1
-ENV CUDA_TOOLKIT_PATH /usr/local/cuda
+ENV TF_CUDA_PATHS "/usr/local/cuda,/usr/lib/x86_64-linux-gnu/"
```

Comment:
Why is `/usr/lib/x86_64-linux-gnu` needed here? It's one of the default path for `ld` (or it should)

Reply:
TF_CUDA_PATHS replaces all the previous path variables, so I merged the values of CUDA_TOOLKIT_PATH and CUDNN_INSTALL_PATH into it.